apps/common/views/layout.html

```diff
@@ -12,13 +12,15 @@
 {{$insideHeader}}{{/insideHeader}}
 
 {{$main}}
-  <main id="content" class="main" role="main">
+  <header id="banner-content">
     <div class="phase-banner">
       <p>
         <strong class="phase-tag">BETA</strong>
         <span>This is a new service – your <a href="mailto:brpformfeedback@homeoffice.gsi.gov.uk?subject=BRP Beta feedback">feedback</a> will help us to improve it.</span>
       </p>
     </div>
+  </header>
+  <main id="content" class="main" role="main">
```

Comment:
Is this good semantic HTML?
Should <header> not go inside <main>?
This page has an example as well as a bit of blurb about ARIA: http://html5doctor.com/the-header-element/